### ISSUE TITLE
Ingest Roden EMSL 2022 FICUS data

### ISSUE THREAD

**Issue by aclum** [microbiomedata/issues#884]
**Deliverable this task is associated with**

_See Deliverables tab here: _

Identifiers
doi 10.46936/fics.proj.2021.60033/60000394
JGI ITS proposal ID 508059
GOLD study ID Gs0160700
EMSL project ID 60033

**RACI**

_Tag people in their roles_
- Responsible:
- Accountable: @aclum @mslarae13 
- Consulted:
- Informed:

**Describe the the task**
- [ ] Ingest from GOLD metagenome and metatranscriptome data
- [ ] link/ingest JGI LC/MS 
- [ ] ingest EMSL NOM, GC/MS, Lipidomics data

**Criteria for completion**
- NMDC study page with multiomics data

**Estimate people time**
- [Hours or days of people time. 1 person, 4 hours]

**Completion Date (Goal)**
- [GOAL!]

**Target Sprint Start & End Dates**
- Start:
- End:

**Tag Blocker/Contingent upon issues**
- [Tag issues]

**Comment by mslarae13** [comment]
Emailed Eric. Will try to get a submission portal entry to get metadata standardized.
If that doesn't happen, we can ingest GOLD and add the additional metadata that was sent to EMSL.
As a note, GOLD has less samples than we received at EMSL. Hoping the user will send GOLD to EMSL map.

**Comment by aclum** [comment]
Per Ben metabolomics hasn't been submitted to MassIVE yet.

**Comment by aclum** [comment]
@mslarae13  any update from Eric?

**Comment by mslarae13** [comment]
No. I can follow up but never responded to my email on the 29th

**Comment by kheal** [comment]
@kheal Tagging myself to keep in the loop re: lipid samples.

**Comment by aclum** [comment]
@mslarae13  Did you hear back from Eric? It has been 4 months so suggest proceeding if we haven't. cc @emileyfadrosh @lamccue
